code/game/objects/items/food/snacks.dm

```diff
@@ -163,3 +163,13 @@
 	tastes = list("pure electricity" = 3, "fitness" = 2)
 	foodtypes = TOXIC
 	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/bluespacebar
+	name = "Bluespace Bar"
+	icon_state = "bluespacebar"
+	desc = "Probably paraphysical, definitely delicious. Orginally designed to help with running through walls."
+	trash_type = /obj/item/trash/bluespacebar
+	food_reagents = list(/datum/reagent/bluespace = 30, /datum/reagent/drug/methamphetamine = 10, /datum/reagent/consumable/nutriment = 5, /datum/reagent/consumable/sugar = 10)
```

Comment:
this is a bit too much reagents for a single bar, even the biggest foods dont have 50 reagents in total.

Reply:
However I can't remove the reagents without making the affect significantly lessened. I tested it many times, with different amounts of each reagent, and this is the balance that gave me the most balanced and fun results. Making it have the same affect with a different amounts of chems would be something beyond my skill level.